pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -416,6 +416,17 @@ async def run(self):
                         lock_id=lock_identifier
                     )
 
+                # publish rhcos on mirror via rhcos_sync job
+                # only if release is EC or a GA release (.0)
+                # job will not mirror & overwrite if destination already exists (sync already happened)
+                # if that is desired, run rhcos_sync with FORCE=true
+                is_ga = assembly_type == AssemblyTypes.STANDARD and self.assembly.endswith(".0")
+                if assembly_type == AssemblyTypes.PREVIEW or is_ga:
```

Comment:
I would prefer to keep it this way :) I like the style of bunching multiple conditions into a bool var, for me it makes the if condition readable, concise and easy to reason with. Unless you think otherwise.